From 2f3f422e507dd5601aa0b0426bc81f59cda020a7 Mon Sep 17 00:00:00 2001
From: Stuart Morgan <smorgan@mythtv.org>
Date: Sun, 7 Sep 2014 12:52:25 +0100
Subject: [PATCH 4/7] MythMusic: Allow smart playlist to be saved even if it
 produces no matches. The nature of smart playlists
 means that it may match tracks in the future.

---
 mythplugins/mythmusic/mythmusic/smartplaylist.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mythplugins/mythmusic/mythmusic/smartplaylist.cpp b/mythplugins/mythmusic/mythmusic/smartplaylist.cpp
index db99b74..0fc546d 100644
--- a/mythplugins/mythmusic/mythmusic/smartplaylist.cpp
+++ b/mythplugins/mythmusic/mythmusic/smartplaylist.cpp
@@ -748,7 +748,7 @@ void SmartPlaylistEditor::updateMatches(void)
 
     m_matchesText->SetText(QString::number(m_matchesCount));
 
-    m_playlistIsValid = (m_matchesCount > 0);
+    m_playlistIsValid = (m_criteriaRows.size() > 0);
     m_showResultsButton->SetEnabled((m_matchesCount > 0));
     titleChanged();
 }
-- 
1.7.10.2

